.travis.yml

```diff
@@ -10,16 +10,29 @@ rvm:
   - 1.9.3
   - 2.0.0
   - 2.1.0
+
 env:
   matrix:
     - PUPPET_GEM_VERSION="~> 3.0.0"
     - PUPPET_GEM_VERSION="~> 3.1.0"
     - PUPPET_GEM_VERSION="~> 3.2.0"
+    - PUPPET_GEM_VERSION="~> 3.3.0"
     - PUPPET_GEM_VERSION="~> 3.4.0"
-    - PUPPET_GEM_VERSION="~> 3.5.1"
+    - PUPPET_GEM_VERSION="~> 3.5.0"
     - PUPPET_GEM_VERSION="~> 3.6.0"
     - PUPPET_GEM_VERSION="~> 3.7.0"
     - PUPPET_GEM_VERSION="~> 3.8.0"
+    - PUPPET_GEM_VERSION="~> 3" PARSER="future"
```

Comment:
You can just set FUTURE_PARSER="yes" and puppetlabs_spec_helper will pick it up and configure it for you. Then you can drop off the change to spec_helper